XCUITests/SyncFAUITests.swift

```diff
@@ -118,141 +118,4 @@ class SyncUITests: BaseTestCase {
         app.navigationBars["Turn on Sync"].buttons["Close"].tap()
         waitForExistence(app.collectionViews.cells["TopSitesCell"])
     }
-
-    // Smoketest
-    /*Disabled due to the new 6 digits authen code to verify account
```

Comment:
Disabled for a long time. Maybe worth exploring adding this test to the sync int tests but not here